README.md

```diff
@@ -632,6 +638,46 @@ Example ESLint configuration:
 }
 ```
 
+### [typescript/member-delimiter-style]
+
+**This rule requires certain options.**
+
+This rule enforces a specific delimiter for interfaces and object types.
+
+Example ESLint configuration:
+
+```json5
+{
+  "rules": {
+    "typescript/member-delimiter-style": [
+      "error",
+      {
+        // If you have `semi: false` in your Prettier config
+        "multiline": {
+          "delimiter": "none"
+        }
+      }
+    ]
+  }
+}
+```
+
+### [typescript/type-annotation-spacing]
```

Comment:
When is this rule useful to have enabled at the same time as using Prettier?

Reply:
⏫